Cython/Compiler/AutoDocTransforms.py

```diff
@@ -54,6 +54,9 @@ def __init__(self, context):
         self.class_name = None
         self.class_node = None
 
+    def _select_format(self, embed, clinic):
+        return embed
```

Comment:
What is this for?

Reply:
That's for a future enhancement I was planning. I should have added a comment about this.

The idea is to support the `__text_signature__` function attribute you get from Python's argument clinic to support `inspect.signature()`. But this would only be useful if setting explicitly the compiler option `binding = False`, otherwise Cython functions work just fine with `inspect.signature()`.

On a second thought, I may very well get rid of this thing. What would be your preference?

Reply:
If it's not needed for this PR, let's leave it out.

Reply:
I removed the method and left some `# clinic: xxx` comments to keep around what should be used for the argument clinic format.